deploy/gk-deploy

```diff
@@ -384,6 +384,11 @@ if [[ ${ABORT} -eq 1 ]]; then
 fi
 
 if [[ ${LOAD} -eq 0 ]]; then
+  check_pods "glusterfs=heketi-pod"
```

Comment:
@raghavendra-talur My Bad. Check pods will abort if it didn't find the pods with these labels. 
I think we can add another flag which can parse in check pods and return appropriate. It is upto you.